ios/brave-ios/Sources/BrowserMenu/BrowserMenu.swift

```diff
@@ -132,10 +132,8 @@ public struct BrowserMenu: View {
             }
           }
           .modifier(MenuRowButtonStyleModifier())
-          .background(
-            Color(braveSystemName: .iosBrowserContainerHighlightIos),
-            in: .rect(cornerRadius: 14, style: .continuous)
-          )
+          .background(Color(braveSystemName: .iosBrowserContainerHighlightIos))
+          .clipShape(.rect(cornerRadius: 14, style: .continuous))
```

Comment:
Small UI bug I noticed that I snuck into it, just ensures the button highlight is clipped to the background shape